operate/client/src/App/ProcessInstance/BottomPanel/VariablePanel/Listeners/styled.ts

```diff
@@ -10,19 +10,21 @@ import {StructuredList as BaseStructuredList} from 'modules/components/Structure
 import {WarningFilled as BaseWarningFilled} from '@carbon/react/icons';
 import {supportError, spacing01, spacing03} from '@carbon/elements';
 import styled from 'styled-components';
-import {Stack} from '@carbon/react';
+import {Stack, Dropdown as BaseDropdown} from '@carbon/react';
 
 const Content = styled.div`
   position: relative;
   height: 100%;
   display: flex;
+  flex-direction: column;
+  padding: var(--cds-spacing-03) var(--cds-spacing-05);
```

Comment:
Carbon's best practice is to use a [stack](https://react.carbondesignsystem.com/?path=/docs/layout-stack--overview) instead of a manual padding. My suggestion would be to either turn `Content` into a `Stack` or add a stack as a parent which then includes `Dropdown` and `StructuredList`.

Reply:
I'm not sure it makes a difference here, as this is an "external" padding, and not between items. It's essentially a margin, but it's gotta be written as padding on the css otherwise it breaks the scrollbar you have a longer list if I remember correctly.

I've tried making `Content` into a `Stack` but I had to manually add that padding to the styles either way. Adding a `Stack` inside `Content` to wrap to the other elements seems redundant to me but since it won't get in the way of anything that's what I did for the sake of consistency with carbon (even though I don't see any `Stack` being used on the other tabs' contents).